Tempura/UITests/UITests.swift

```diff
@@ -31,14 +31,14 @@ public enum UITests {
    */
   
   
-  public struct VCScreenSnapshot<VC: AnyViewController> {
+  public struct VCScreenSnapshot<VC: UIViewController, V: UIView> {
```

Comment:
If we do this we are not able to constrain V to be the rootView of VC.
If we want to test simple UIViewControllers it is ok to lose this but we don't want that for our `ViewController`s.
I suggest creating a different class to test simple `UIViewController`s.